setup.py

```diff
@@ -11,7 +11,7 @@
 
 setuptools.setup(
     name="nbresuse",
-    version="0.3.6",
+    version="0.4.0",
```

Comment:
Should this be part of a separate release commit?

Reply:
It is a separate commit, just in the same PR. Should probably be ok?

Reply:
Thinking that we might want to add a couple more changes between this PR and the release.

For example moving the lab extension here to the nbresuse repo: https://github.com/jupyterlab/jupyterlab/issues/9363

Reply:
@jtpio hmm, ideally I'd like to do a quick release here, and then maybe do a 0.5 with other changes - so only change in 0.4 is the deprecation. Since existing JupyterLab installations will work with 0.4 anyway, it should be probably ok?

Reply:
Sounds good :+1: 

So to summarize:

- 0.4.0 with the soft-deprecated endpoint. Users in lab 2.x will still see the indicator because `/metrics` is still available
- if the status bar item is removed from core for lab 3.0, then we release nbresuse 0.5.0 with this change: https://github.com/yuvipanda/nbresuse/pull/69. With 0.5.0 removing the `/metrics` endpoint so users with the lab 2.x + nbresuse combination don't see the indicator twice in the status bar